src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -216,6 +229,10 @@ public static function getContacts(array $owner, array $rel, string $module, int
 			$data['orderedItems'] = $list;
 		}
 
+		if (!empty($cachekey)) {
```

Comment:
According to the description of the parameter, I believe you should have checked the value of `$nocache` here.